app/ts/SharedUI/Spinner.tsx

```diff
@@ -0,0 +1,21 @@
+export function Spinner() {
+	return <svg style = { { height: '1em' } } class = 'spinner' viewBox = "0 0 100 100" xmlns = "http://www.w3.org/2000/svg"><circle cx = "50" cy = "50" r = "45"/></svg>
+}
+
+export function BigSpinner() {
+	return <svg style = { { height: '2em' } } class = 'spinner' viewBox = "0 0 100 100" xmlns = "http://www.w3.org/2000/svg"><circle cx = "50" cy = "50" r = "45"/></svg>
+}
+
+export function CenteredBigSpinner() {
+	return <div style = 'max-width: fit-content; margin-inline: auto; padding: 20px;'>
+		<BigSpinner/>
+	</div>
+}
+
+export function PageLoadingSpinner() {
+	return <div class = 'subApplication'>
+		<section class = 'subApplication-card'>
+			<CenteredBigSpinner/>
+		</section>
+	</div>
+}
```

Comment:
I feel like for at least `Spinner` and `BigSpinner`, you could just have one and parameterize it.  You could also parameterize the centering pretty easily I believe.  Page loading seems a bit more complicated, and perhaps worthy of being its own spinner.